docs/notebooks/inequality_constraints.pct.py

```diff
@@ -144,7 +152,7 @@ def create_bo_model(data):
 if result.error is not None: raise result.error
 
 # %% [markdown]
-# To conclude, we visualise the resulting data. Orange dots show the new points queried during optimization. Notice the concentration of these points in regions near the local minima.
+# Finally, we visualise the resulting data. Orange dots show the new points queried during optimization. Notice the concentration of these points in regions near the local minima.
```

Comment:
At last!

Reply:
but anyway, why the change? finally sounds like we've been waiting the whole notebook for this